internal/checks/checks.go

```diff
@@ -382,6 +388,14 @@ func (c *Updater) loop(ctx context.Context) (bool, error) {
 
 	c.probe = &result.Probe
 
+	// usageReporter will be nil if the usage report configuration is disabled
+	if c.usageReporter != nil {
+		err = c.usageReporter.ReportProbe(context.Background(), result.Probe)
```

Comment:
Should we pass the `ctx` that is passed to the `loop()` method instead?

Reply:
Let me try it locally and just validate. I can't remember if I had a distinct reason why I didn't, but as long as it's working I don't see the need to create a new ctx.